File: CPP/Program/Manajer.cpp


class Manajer : public AnggotaTim {
private:
    string _divisi, _levelJabatan;
    int _jumlahMenang;
public:
    Manajer(string id = "", string nama = "", string nickname = "", string divisi = "", string level = "", int menang = 0)
        : AnggotaTim(id, nama, nickname) {
        this->_divisi = divisi; this->_levelJabatan = level; this->_jumlahMenang = menang;
    }
    string getDivisi() const { return this->_divisi; }
    string getLevelJabatan() const { return this->_levelJabatan; }
    int getJumlahMenang() const { return this->_jumlahMenang; }
    ~Manajer() {}
};

File: Java/Program/Manajer.java
public class Manajer extends AnggotaTim {
    private String divisi;
    private String levelJabatan;
    private int jumlahMenang;

    public Manajer(String id, String nama, String nickname, String divisi, String level, int menang) {
        super(id, nama, nickname);
        this.divisi = divisi;
        this.levelJabatan = level;
        this.jumlahMenang = menang;
    }

    public String getDivisi() { return this.divisi; }
    public String getLevelJabatan() { return this.levelJabatan; }
    public int getJumlahMenang() { return this.jumlahMenang; }
}